### DISCUSSION TITLE
Bolt with LED-Ring.. Help!

### DISCUSSION THREAD

**Discussion by Scicrew** [bitcraze/discussions#954] [Q&A]
Hello..
We have built a similar setup in the [Bolt](https://www.bitcraze.io/2020/10/testing-crazyflie-bolt-and-1-cell-li-ion/) of the Bolt with 1S battery (18650) and Lighthouse Deck. Up to this stage we did not install the [LED-Ring deck](https://www.bitcraze.io/products/led-ring-deck/) yet.
Then we have tested our built Bolt with Skybrush Studio, Server and Live and we had a successful flight with no issues.

**The problem** started when we installed the LED-Ring deck at the bottom of the bolt (by creating longer "Male long deck connectors"). We made a test show via Skybrush but It seems there is not enough current to feed both the Bolt and LED-ring Deck (in other words Onboard 5V DC/DC could not feed enough current for both the LED and Bolt). The LEDs were not bright enough and the flight was unstable. This was very frustrating since we believe we did everything right.

**Our questions**:
- Is there a way to resolve this issue with the current above mention built (1S Bolt (18650) + Lighthouse Deck + LED-Ring Deck)? 
- If we go with Bolt with 2S battery (7.4 V) (either 2 X 18650 batteries or 2S Li-Po battery), will this resolve the issue? and how to do the connections to ensure there is enough current for smooth flight and bright LEDs?

**Comment by tobbeanton** [comment]
> Is there a way to resolve this issue with the current above mention built (1S Bolt (18650) + Lighthouse Deck + LED-Ring Deck)?

There is a hack you can do described in [this](https://github.com/orgs/bitcraze/discussions/906) discussion.

> If we go with Bolt with 2S battery (7.4 V) (either 2 X 18650 batteries or 2S Li-Po battery), will this resolve the issue? and how to do the connections to ensure there is enough current for smooth flight and bright LEDs?

Running it in 2S solves the problem because it is the input current that can not go much higher then 500mA (DC/DC) limitation.
